website/src/components/TestimonialCarousel.tsx

```diff
@@ -45,7 +45,7 @@ export const TestimonialCarousel: React.FunctionComponent<Props> = ({
                     <img src="/case-studies/quote.svg" className="testimonial-carousel__quote-icon mb-5" alt="" />
                     <Carousel controls={false}>
                         {testimonials.map(({ customer, logo, quote, author, cta }, i) => (
-                            <div className={`${slugify(customer).toLowerCase()} testimonial-carousel__testimonial`}>
+                            <div key={`testimonial-${i}`} className={`${slugify(customer).toLowerCase()} testimonial-carousel__testimonial`}>
```

Comment:
What is the intention behind adding `key` here?

Reply:
I believe react likes every element generated inside of loops to have unique keys (I get a console error when viewing this page locally). See https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
Good to know, thanks!